### ISSUE TITLE
Qidi X Smart 3

### ISSUE THREAD

**Issue by mendeste** [leadustin/QIDI-up2date-english#13]
Hi Leadustin
great work well done
Would you have the Printer.cfg and macro for Qidi X Smart 3
it would save me alot time 
if not I'll need to use the current one and clean it up

Cheers

**Comment by leadustin** [comment]
> Hi Leadustin great work well done Would you have the Printer.cfg and macro for Qidi X Smart 3 it would save me alot time if not I'll need to use the current one and clean it up
> 
> Cheers

Hello mendeste,
I currently have nothing for the Smart 3. I would have a look at the weekend to see if I can do something about it

**Comment by mendeste** [comment]
Thank you
You are a legend

Regards

**Comment by leadustin** [comment]
Just out of curiosity, have you flashed your Smart 3 to Klipper 0.12 etc? Because if not, a new printer.cfg and the macros won't help much.

**Comment by mendeste** [comment]
Hi Leadustin
not yet waiting on 32g emmc to arrive should be this week
then let the fun begin :)
Regards

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by mendeste** [comment]
Hi
I've also got spare Extruder board
So was going to keep the original board and emmc
If anything went wrong I could always go back
That's my backup plan 😀

**Comment by mendeste** [comment]
Sorry
Also got K-Touch was going to use this as replacement screen

**Comment by leadustin** [comment]
Good luck with that. Remember that I have written this guide primarily for the Max-3 and Plus-3. However, you will find a customised printer.cfg and the macros folder in the download. The mainboard of the Smart 3 is identical to the other two printers. Only the chamber heater is missing. If you have any questions, please write

**Comment by mendeste** [comment]
Thank you 
will let you know how i go :)

**Comment by mendeste** [comment]
Hi Leadustin
Everything installed used Fluidd instead of mainsail 
Just got one problem, when I home it should home X91 and Y91, which the center of the build plate 
But it homing to far down and across 
Regards

**Comment by mendeste** [comment]
Thank you
I'm bet confused get this from where /home/mks/fluidd-config/client.cfg ( Is this my original client.cfg file?
Regards

**Comment by leadustin** [comment]
> Thank you I'm bet confused get this from where /home/mks/fluidd-config/client.cfg ( Is this my original client.cfg file? Regards

Yes, this is the original client.cfg. I have packed the one from Mainsail into the macros folder and included it in printer.cfg.

**Comment by mendeste** [comment]
Hi Leadustin
All working great after 4 attempts 🤪
Tried twice with fluidd but moonraker would not connect after reboot. So installed with mainsail config everything and need to comment out some entries.
Then once everything was working, I installed Fluidd using kiauh. Works great side by side with mainsail.
I only have one problem this come up while printing but does not effect the printer
Stepper driver 'stepper_y' is over-heating 
But it’s not, same temperature as the other 2 stepper motors

**Comment by leadustin** [comment]
> Hi Leadustin All working great after 4 attempts 🤪 Tried twice with fluidd but moonraker would not connect after reboot. So installed with mainsail config everything and need to comment out some entries. Then once everything was working, I installed Fluidd using kiauh. Works great side by side with mainsail. I only have one problem this come up while printing but does not effect the printer Stepper driver 'stepper_y' is over-heating But it’s not, same temperature as the other 2 stepper motors

I'm glad to hear that your conversion went well. However, I can't really help you with the temperature problem. It may be a cooling problem. Or the stepper has a hardware problem. Have you installed Klipper TMC Autotune? This can cause problems under certain circumstances.

**Comment by mendeste** [comment]
Hi Leadustin

All good working great
The temperature problem my fault forgot to plug in the mother board Fan
See attached pictures
Working with BBT K-Touch screen

**Comment by mendeste** [comment]
![IMG_1766](https://github.com/user-attachments/assets/50ec9cfc-e21f-4f61-a228-c2219a426cea)
![IMG_1765](https://github.com/user-attachments/assets/6dfe13e5-ecff-494f-a80e-723bcb1ab85f)

**Comment by leadustin** [comment]
looks very good. Good job

**Comment by theclue** [comment]
Hi all,

I would also like to make the conversion to my x-Smart 3, I was just wondering if you perhaps can make an easier step-by-step sum-up of your debugging results about the homing issue. While trying to follow-up the thread, it's not totally clear to me I must admit.

Additionally, in your opinion, is the QIDI PSU powerful enough to power the Rpi3 used for alternative display (using a step-down ofc)? I would love a "all-in-case" solution for the conversion :)

**Comment by mendeste** [comment]
Hi leadustin
Just update everything going well
Even added nozzle wipe see picture
![IMG_1767](https://github.com/user-attachments/assets/162bf2f3-0566-4c33-be29-f2769288ba94)

**Comment by theclue** [comment]
Howdy :)

I was wondering if you have some updated about the homing issue and about the Wifi. In other issues it seems Wifi won't work and I have not LAN on my setup actually...
